### DISCUSSION TITLE
Voice cloning in other language leads to not recognized patterns

### DISCUSSION THREAD

**Discussion by a-klos** [serp-ai/bark-with-voice-clone#76] [Q&A]
We are cloning a voice in another language. The results are good, but the patterns, like [laughs] etc produce not the expected output. Is there any reason for that? Did someone face a similar behavior?

**Comment by saichand018** [comment]
can you suggest how can we clone for other lanuages ?

**Comment by devinschumacher** [comment]
We just launched our follow up to BARK -- an AI Voice Cloner Desktop App (runs locally, offline, super realistic voice cloning, data privacy, no coding needed)

👉Check it out: https://www.youtube.com/watch?v=84JVGtkapik

@saichand018  @a-klos
